api/src/pcapi/scripts/venue/venue_ban_id/add_ban_id_to_all_permanent_venues.py

```diff
@@ -0,0 +1,100 @@
+import logging
+import pathlib
+
+import requests
+
+from pcapi.core.offerers.models import Venue
+from pcapi.repository import repository
+
+
+logger = logging.getLogger(__name__)
+PATH = pathlib.Path(__file__).parent.resolve()
+CHUNK_SIZE = 1000
+TIMEOUT = 3
+
+
+def request_adresse_api(
+    filename: pathlib.Path, permanent_venues: list[Venue], already_requested_venues_ids: list
+) -> None:
+    # Call to API
+    base_url = "https://api-adresse.data.gouv.fr/search/csv/"
+    header = "Venue ID,adresse,postcode,city\n"
+    batches = [permanent_venues[i : i + CHUNK_SIZE] for i in range(0, len(permanent_venues), CHUNK_SIZE)]
+    for i, batch in enumerate(batches, start=1):
+        payload = f"{header}"
+        for venue in batch:
+            if venue.id not in already_requested_venues_ids:
+                payload += f"{venue.id},{venue.address},{venue.postalCode},{venue.city}\n"
+
+        if payload == header:
+            logger.info("No venues to request from Adresse API, please check or delete %s", filename)
+            return
+
+        files = [
+            ("data", payload),
+            ("result_columns", (None, "result_id")),
+        ]
+
+        response = requests.post(base_url, files=files, timeout=TIMEOUT)  # type: ignore [arg-type]
+        if response.status_code == 200:
+            # Save response to csv file
+            result_data = csv.DictReader(StringIO(response.text))
+            fieldnames = result_data.fieldnames if result_data.fieldnames else []
+            output_exists = pathlib.Path(filename).is_file()
+            # If existing, append to file
+            with open(filename, mode="a", newline="", encoding="utf-8") as csv_output:
+                writer = csv.DictWriter(csv_output, fieldnames=fieldnames)
+                if not output_exists:
+                    writer.writeheader()
+                writer.writerows(result_data)
+        else:
+            logger.info("Request wasn't successful. Status code is %s", {response.status_code})
+
+
+def update_venues_from_csv(filename: pathlib.Path, permanent_venues: list[Venue]) -> None:
+    # Read result csv and update banId
+    with open(filename, "r", encoding="utf-8") as file:
+        csv_reader = csv.DictReader(file)
+        permanent_venue_ids = {v.id for v in permanent_venues}
+        for data in csv_reader:
+            venue_id = int(data.get("Venue ID", -1))
+            if venue_id in permanent_venue_ids:
+                venue = next(v for v in permanent_venues if v.id == venue_id)
+                venue.banId = data.get("result_id", None)
+    # Log infos of which venues have been updated
+    venues_with_ban_id = [v for v in permanent_venues if v.banId]
+    venues_without_ban_id = [v for v in permanent_venues if not v.banId]
+    logger.info("Venue IDs where banId have been updated %s", [v.id for v in venues_with_ban_id])
+    logger.info("Venues where no banId has been found %s", [v.id for v in venues_without_ban_id])
```

Comment:
[détail] ~Ces 2 lignes vont logger plusieurs milliers d'identifiants. Il y a une taille limite pour les logs, est-ce que ça ne risque pas de dépasser ?~ (Apparemment la limite est à 256 Ko, donc on pourrait largement logger 15_000 éléments.)

D'autre part, y a-t-il vraiment un intérêt à logger, ici ? On pourra facilement retrouver ces nombres en faisant un `select count(*) from venue where .... and "banId" is null`, non ?